lnrpc/rpc.proto

```diff
@@ -846,6 +846,17 @@ service Lightning {
             body: "*"
         };
     };
+
+    /** lncli: `listsweeps`
+    ListSweeps returns a list of the sweep transactions our node has produced.
+    Note that these sweeps may not be confirmed yet, as we record sweeps on
+    broadcast, not confirmation.
+    */
+    rpc ListSweeps (ListSweepsRequest) returns (ListSweepsResponse) {
```

Comment:
Perhaps this should go to the wallet kit sub-server instead? It already has a `PendingSweeps` endpoint.

Reply:
Moved into wallet 👍